Fix:

error: invalid use of incomplete typedef 'WINDOW' {aka 'struct _win_st'} (gcc)
error: incomplete definition of type 'struct _win_st' (clang)

https://sourceforge.net/p/aa-project/patches/10/
--- src/aacurses.c.orig	2001-04-26 09:37:31.000000000 -0500
+++ src/aacurses.c	2024-11-10 03:33:33.000000000 -0600
@@ -71,8 +71,7 @@
 {
     if (__resized_curses)
 	curses_uninit(c), curses_init(&c->params, NULL,&c->driverparams, NULL), __resized_curses = 0;
-    *width = stdscr->_maxx + 1;
-    *height = stdscr->_maxy + 1;
+    getmaxyx(stdscr, *height, *width);
 #ifdef GPM_MOUSEDRIVER
     gpm_mx = *width;
     gpm_my = *height;
